packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -11,78 +11,114 @@
  */
 
 import {ActionButton} from '@react-spectrum/button';
+import {Flex} from '@react-spectrum/layout';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 import {Tooltip, TooltipTrigger} from '../src';
 
 storiesOf('TooltipTrigger', module)
   .add(
     'default',
-    () => render('This is a tooltip.', {})
-  ).add(
+    () => render({})
+  )
+  .add(
     'placement: left',
-    () => render('This is a tooltip.', {placement: 'start'})
-  ).add(
+    () => render({placement: 'left'})
+  )
+  .add(
+    'placement: start',
+    () => render({placement: 'start'})
+  )
+  .add(
     'placement: top',
-    () => render('This is a tooltip.', {placement: 'top'})
-  ).add(
+    () => render({placement: 'top'})
+  )
+  .add(
     'placement: bottom',
-    () => render('This is a tooltip.', {placement: 'bottom'})
-  ).add(
+    () => render({placement: 'bottom'})
+  )
+  .add(
     'isDisabled',
-    () => render('This is a tooltip.', {placement: 'left', isDisabled: true})
-  ).add(
-     'multiple tooltips',
-     () => renderMultipleTriggers('This is a tooltip.', {placement: 'left'})
-   );
+    () => render({placement: 'start', isDisabled: true})
+  )
+  .add(
+    'delay',
+    () => render({delay: true})
+  )
+  .add(
+    'multiple tooltips',
+    () => renderMultipleTriggers({placement: 'start'})
+  )
+  .add(
+    'delay multiple tooltips',
+    () => renderMultipleTriggers({delay: true})
+  )
+  .add(
+    'isOpen',
+    () => renderMultipleTriggers({isOpen: true})
```

Comment:
hmm, I don't think this should be allowed. Only one tooltip in an application should ever be displayed at a time.

Reply:
So if we don't allow a controlled state, do we want to still have onOpenChange?
I think I'd argue yes, that way metrics can be collected

Reply:
mm.... this is going to be unfortunate for chromatic :(

Reply:
I guess I could specify a delay of 0 and autofocus, but then we can only do one per snapshot